index.html

```diff
@@ -1,6 +1,8 @@
 <!doctype html>
 <html>
   <head>
+  <!-- This edit here with the title tag is for the tab name. -->
+  <title>This is tab</title>
```

Comment:
Here's where you'd add the stylesheet and how: `<link href="styles.css" rel="stylesheet" type="text/css">`